server/repository/src/db_diesel/changelog/test.rs

```diff
@@ -26,12 +34,12 @@ async fn test_changelog() {
     // single entry:
     location_repo.upsert_one(&mock_location_1()).unwrap();
     let mut result = repo.changelogs(0, 10, None).unwrap();
-    assert_eq!(1, result.len());
+    assert_eq!(2, result.len());
```

Comment:
> Can this be moved to another issue?

Would prefer if the diff is reduced here, rather then adding more in the new branch, you can just commit this: https://github.com/openmsupply/open-msupply/compare/1248-Push-name-changes-to-central-server-...1248-example